django_project/geosight/data/serializer/related_table.py

```diff
@@ -40,7 +40,11 @@ def get_creator(self, obj: RelatedTable):  # noqa: D102
         return obj.creator.get_full_name() if obj.creator else None
 
     def get_fields_definition(self, obj: RelatedTable):  # noqa: D102
-        return obj.fields_definition
+        return [{
```

Comment:
Not really happy about this, but trying to decouple the model `RelatedTable.fields_definition` from the API. I'm afraid if I put this into the `RelatedTableFieldSerializer` it might break other parts of the codebase since it's what's used internally in `RelatedTable.fields_definition()`?

Reply:
hi @victorzinho 
it seems you rename 'alias' to 'label'
You can change it on serializer @victorzinho 
Because alias or label is not being used by frontend for now

Reply:
Done. Implemented a new serializer since it was going to be needed anyway for the POST endpoint: https://github.com/unicef-drp/GeoSight-OS/pull/212